[BUG] PhpStorm exception on Windows with v1.3.2

## 📚 Before Submitting

**Please check our [documentation](https://genie.devoxx.com/docs/getting-started/introduction) first!**

You may also find helpful information in our [Features Overview](https://genie.devoxx.com/docs/features/overview) or search [existing issues](https://github.com/devoxx/DevoxxGenieIDEAPlugin/issues).

## Bug Description

**Describe the bug:**
On PhpStorm 2025.3.3.  I always get the following Exception: 

com.intellij.diagnostic.PluginException: com/intellij/openapi/compiler/CompilerTopics [Plugin: com.devoxx.genie]
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:517)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.NoClassDefFoundError: com/intellij/openapi/compiler/CompilerTopics
	at com.devoxx.genie.service.PostStartupActivity.registerEventAutomationListeners(PostStartupActivity.java:81)
	at com.devoxx.genie.service.PostStartupActivity.execute(PostStartupActivity.java:59)
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:511)
	... 7 more
Caused by: java.lang.ClassNotFoundException: com.intellij.openapi.compiler.CompilerTopics PluginClassLoader(plugin=PluginMainDescriptor(name=DevoxxGenie, id=com.devoxx.genie, version=1.3.2, isBundled=false, path=~\AppData\Roaming\JetBrains\PhpStorm2025.3\plugins\DevoxxGenie), packagePrefix=null, state=active, parents=ContentModuleDescriptor(id=intellij.symfony, package=com.intellij.symfony) <- PluginMainDescriptor(name=PHP, id=com.jetbrains.php, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\php-impl), ContentModuleDescriptor(id=intellij.php.completionMLRanking, package=com.intellij.php.completionMLRanking) <- PluginMainDescriptor(name=PHP, id=com.jetbrains.php, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\php-impl), ContentModuleDescriptor(id=intellij.javascript.sh, package=com.intellij.lang.javascript.sh) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.backend/diagrams, package=com.intellij.lang.javascript.modules.diagram, descriptorPath=intellij.javascript.backend.diagrams.xml) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.sharedIndexes.generator, package=com.intellij.lang.javascript.indexing.shared.generator) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.sharedIndexes, package=com.intellij.lang.javascript.indexing.shared) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.devkit, package=com.intellij.lang.javascript.devkit) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.parser, loadingRule=EMBEDDED, package=com.intellij.javascript.parser) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.java, package=com.intellij.javascript.java) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.aqua.runners.playwright.js, package=com.intellij.aqua.runners.playwright.js) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.aqua.runners.js.core, package=com.intellij.aqua.runners.js.core) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.aqua.runners.cypress, package=com.intellij.aqua.runners.cypress) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.common, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.platform.vcs.log.graph) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.relaxng) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.vcs.log) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.vcs.dvcs) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.protocolReaderRuntime) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.backend) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.ui) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.tasks) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools.auth.base) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools.auth) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.vcs.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.vcs.log.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.vcs.dvcs.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.spellchecker) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.spellchecker.xml) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.tasks.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.javascript.psi.impl, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.common.css, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.frontend) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), PluginMainDescriptor(name=PHP, id=com.jetbrains.php, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\php-impl), )
	... 10 more


## Steps to Reproduce

Start ide with plugin 1.3.2


## Screenshots/Logs

com.intellij.diagnostic.PluginException: com/intellij/openapi/compiler/CompilerTopics [Plugin: com.devoxx.genie]
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:517)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.NoClassDefFoundError: com/intellij/openapi/compiler/CompilerTopics
	at com.devoxx.genie.service.PostStartupActivity.registerEventAutomationListeners(PostStartupActivity.java:81)
	at com.devoxx.genie.service.PostStartupActivity.execute(PostStartupActivity.java:59)
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:511)
	... 7 more
Caused by: java.lang.ClassNotFoundException: com.intellij.openapi.compiler.CompilerTopics PluginClassLoader(plugin=PluginMainDescriptor(name=DevoxxGenie, id=com.devoxx.genie, version=1.3.2, isBundled=false, path=~\AppData\Roaming\JetBrains\PhpStorm2025.3\plugins\DevoxxGenie), packagePrefix=null, state=active, parents=ContentModuleDescriptor(id=intellij.symfony, package=com.intellij.symfony) <- PluginMainDescriptor(name=PHP, id=com.jetbrains.php, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\php-impl), ContentModuleDescriptor(id=intellij.php.completionMLRanking, package=com.intellij.php.completionMLRanking) <- PluginMainDescriptor(name=PHP, id=com.jetbrains.php, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\php-impl), ContentModuleDescriptor(id=intellij.javascript.sh, package=com.intellij.lang.javascript.sh) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.backend/diagrams, package=com.intellij.lang.javascript.modules.diagram, descriptorPath=intellij.javascript.backend.diagrams.xml) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.sharedIndexes.generator, package=com.intellij.lang.javascript.indexing.shared.generator) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.sharedIndexes, package=com.intellij.lang.javascript.indexing.shared) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.devkit, package=com.intellij.lang.javascript.devkit) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.parser, loadingRule=EMBEDDED, package=com.intellij.javascript.parser) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.java, package=com.intellij.javascript.java) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.aqua.runners.playwright.js, package=com.intellij.aqua.runners.playwright.js) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.aqua.runners.js.core, package=com.intellij.aqua.runners.js.core) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.aqua.runners.cypress, package=com.intellij.aqua.runners.cypress) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.common, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.platform.vcs.log.graph) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.relaxng) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.vcs.log) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.vcs.dvcs) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.protocolReaderRuntime) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.backend) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.ui) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.tasks) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools.auth.base) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools.auth) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.vcs.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.vcs.log.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.vcs.dvcs.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.spellchecker) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.spellchecker.xml) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.platform.tasks.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\lib), ContentModuleDescriptor(id=intellij.javascript.psi.impl, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.common.css, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), ContentModuleDescriptor(id=intellij.javascript.frontend) <- PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), PluginMainDescriptor(name=JavaScript and TypeScript, id=JavaScript, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\javascript-plugin), PluginMainDescriptor(name=PHP, id=com.jetbrains.php, version=253.31033.138, isBundled=true, path=C:\Program Files\JetBrains\PhpStorm 2025.3.3\plugins\php-impl), )
	... 10 more


## Environment

- **IntelliJ IDEA Version**: phpStorm 2ß25.3.3
- **Plugin Version**: 1.3.2 [e.g. 0.9.2]
- **Operating System**:  Windows 11
- **LLM Provider**: LM Studio
- **Model**: Vaious

## Additional Context

problems started after update to 1.2., had to implement rendering hardware workaround from FAQ. Afterwards came this problem worked ok before.

--- a/src/main/java/com/devoxx/genie/service/PostStartupActivity.java
+++ b/src/main/java/com/devoxx/genie/service/PostStartupActivity.java
@@ -66,28 +66,30 @@ public class PostStartupActivity implements ProjectActivity {
      * Listeners are tied to the project lifecycle and disposed when the project closes.
      */
     private void registerEventAutomationListeners(@NotNull Project project) {
+        MessageBusConnection connection = project.getMessageBus().connect();
+
+        // File editor events (FILE_OPENED)
+        connection.subscribe(FileEditorManagerListener.FILE_EDITOR_MANAGER,
+                new FileEventListener(project));
+
+        // File save events (FILE_SAVED) — application-level topic
+        connection.subscribe(FileDocumentManagerListener.TOPIC,
+                new FileSaveListener());
+
+        // Build/compilation events (BUILD_FAILED, BUILD_SUCCEEDED)
+        // CompilerTopics is only available in IDEs with compiler support (e.g. IntelliJ IDEA),
+        // not in PhpStorm, WebStorm, etc.
         try {
-            MessageBusConnection connection = project.getMessageBus().connect();
-
-            // File editor events (FILE_OPENED)
-            connection.subscribe(FileEditorManagerListener.FILE_EDITOR_MANAGER,
-                    new FileEventListener(project));
-
-            // File save events (FILE_SAVED) — application-level topic
-            connection.subscribe(FileDocumentManagerListener.TOPIC,
-                    new FileSaveListener());
-
-            // Build/compilation events (BUILD_FAILED, BUILD_SUCCEEDED)
             connection.subscribe(CompilerTopics.COMPILATION_STATUS,
                     new BuildCompilationListener(project));
-
-            // Process exit events (PROCESS_CRASHED)
-            connection.subscribe(ExecutionManager.EXECUTION_TOPIC,
-                    new ProcessExitListener());
-
-            log.debug("Registered event automation listeners for project: {}", project.getName());
-        } catch (Exception e) {
-            log.warn("Failed to register some event automation listeners (IDE may lack required APIs)", e);
+        } catch (NoClassDefFoundError e) {
+            log.debug("CompilerTopics not available in this IDE, skipping build compilation listener");
         }
+
+        // Process exit events (PROCESS_CRASHED)
+        connection.subscribe(ExecutionManager.EXECUTION_TOPIC,
+                new ProcessExitListener());
+
+        log.debug("Registered event automation listeners for project: {}", project.getName());
     }
 }

